README.md

```diff
@@ -11,47 +11,116 @@ ADaM in R Asset Library
+* Vignettes for ADaM dataset specific functionality (i.e. dictionary coding, date imputation,
+SMQs ...)
+
+## Types of Packages
+There will be 3 foreseeable types of `{admiral}` packages:
+* Core package - one package containing all core functions required to create ADaMs,
+usable by any company (i.e. general derivations, utility functions and checks for ADSL, OCCDS and BDS)
+* TA (Therapeutic Area) package extensions - one package per TA with functions that are
+specific to algorithms and requirements for that particular TA (e.g. `{admiralonco}`)
+* Company package extensions - specific needs and plug-ins for the company, such as access to metadata
+(e.g. `{admiralroche}` or `{admiralgsk}`)
+
+## Admiral Manifesto
+For `{admiral}` and all extension packages, we prioritise providing our users with a **simple to adopt** toolkit
+that enables them to produce **readable** and **easily constructible** ADaM programs. The following explains
+our philosophy, which we try to adhere to across the `{admiral}` family of packages.
+There isn’t always a clear single, straight forward rule but there are guiding principles we adhere to.
+This manifesto helps show the considerations of our developers when making decisions.
+
+We have four priority design principles to achieve the main goal as follows:
+
+_**Usability**_
+
+All `{admiral}` functions should be easy to use. 
+* Documentation is an absolute priority. Each function reference page should cover the purpose,
+descriptions of each argument with permitted values, the expected input and output, with clear real-life
+examples - so that users don’t need to dig through code to find answers.
+* Vignettes that complement the functional documentation to help users see how best the functions can be
+applied to achieve customary ADaM requirements.
+* Functions should be written and structured in a way that users are able to read, re-use or extend them
+for study specific purposes if needed (see Readability below).
+
+_**Simplicity**_
+
+Each function has a clear purpose.
+* We try not to ever design single functions that could achieve numerous very different derivations, as for
+example if you as a user pick up a function with >10 different arguments then chances are it is going to be
+difficult to understand if this function could be applied for your specific need. The intention is that
+arguments/parameters can influence how the output of a function is calculated, but not change the purpose of
+the function.
+* On the other hand, we do try to combine similar tasks and algorithms into one function to reduce the amount
+of repetitive, similar functionality (e.g. one study day calculation rather than a function per variable).
+Importantly here though, functions cannot be too general and trying to fulfil multiple, complex purposes.
+Similarly, functions should not allow expressions as arguments that are used as code snippets in function calls.
```

Comment:
```suggestion
* We try to combine similar tasks and algorithms into one function to reduce the amount of repetitiveness and/or group together similar functionality (e.g. one study day calculation rather than a function per variable).
* We strive to design functions that are not too general and trying to fulfil multiple, complex purposes.
* Functions should not allow expressions as arguments that are used as code snippets in function calls.
```

Reply:
...this is hard.  I feel like declarative sentences are really important for a manifesto.  Some of these statements seem too wordy.

Reply:
I see your point. Let me go through it once more.

Reply:
@bms63 would that be better?